examples/predict_video_on_local.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+import logging
+
+from geti_sdk.demos import predict_video_on_local
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
```

Comment:
Could you add a brief description for the script? Can be done by passing `description=...` to the ArgumentParser